### DISCUSSION TITLE
是否考虑支持新建会话包括选定插件?

### DISCUSSION THREAD

**Discussion by timiil** [linweiyuan/go-chatgpt-api#145] [Ideas]
如题，如果账户已经使能插件并且安装了插件，我们执行创建会话的时候，是否可以加入pluginIDs:[ ... ] ？

**Comment by linweiyuan** [comment]
问得好

我不是 Plus 账号（理论上未来也不会开，没有这个刚需），申请了也还没有通过，无法测试

你可以考虑下提 PR

**Comment by linweiyuan** [comment]
#169
